src/assets/js/index.js

```diff
@@ -18,14 +18,23 @@ midaas.indexPage = {
     // ]];
 
     // fetch data remotely...
-    $.getJSON( "https://x0suepap69.execute-api.us-east-1.amazonaws.com/development/income/quantiles", function( data ) {
-      dataArr = [];
-      for (var d in data) {
-        dataArr.push(data[d]);
-      }
-      midaas.chart.createChart(labels, [dataArr]);
-    });
+    var dataArr = [],
+        url =  "https://x0suepap69.execute-api.us-east-1.amazonaws.com/development/income/quantiles";
 
+    $.ajax({
+        dataType: "json",
+        url: url,
+        data: function(data) {
+          for (var d in data) {
```

Comment:
The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.